default/customizations/custom_sitreps.txt

```diff
@@ -105,6 +105,51 @@ EffectsGroup
             empire = Source.Owner
 
 
+EffectsGroup
+    scope = And [
+        System
+        // The following assets need protection
+        Contains And [
+            OwnedBy empire = Source.Owner
+            Or [
+                Planet
+                Ship
+            ]
+        ]
+        // Dont warn about system where there are already enemies this turn
```

Comment:
Do you want to never warn if there are any enemies in the system... or:
-Enemies with the same owner as those arriving
-Enemies visible to the owner the recipient of the message

Reply:
The warning is to draw attention to the system. If there was already combat, you have already a message for doing that.

I didnt know that i have to script a visibility check. If I remember correctly, without OwnedBy empire = Source.Owner i did not get matches outside my known empire without checking visibility, is that the way it works? Does not sound right.

Reply:
I don't understand your question.

Effects run on the server based on the actual universe, not the universe as seen by a particular player. If you don't add a visibility test to some effect, it won't add it automatically.

Reply:
That makes more sense to me. So probably I had some other condition which limited the results. If not there should have been a lot of sitreps about system i did not know of yet.

Reply:
Not about systems your empire can't see, but about ships that aren't visible to the player's empire that are arriving in systems you can see. The scope already limits it to systems containing objects owned by the empire receiving the sitrep messages.